dev/ci/scripts/utils/find_homegfs.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+"""
+Utility module to find the HOMEgfs (repository root) directory by traversing up
+the file system until finding a directory that contains the .github subdirectory.
+"""
+
+import os
+from pathlib import Path
+
+
+def find_homegfs(start_path=None):
```

Comment:
we could consider a method e.g. `find_upward(search_for: str, start_path: Path = None)` as part of the `wxflow`.  It would then be used as:

```python
from wxflow import find_upward

homegfs_ = find_upward('.github', start_path=Path(os.getcwd()))
```

Reply:
Ok, let's do that latter.

Reply:
Opened https://github.com/NOAA-EMC/wxflow/issues/51.  Resolving this conversation.